mono/utils/mono-publib.h

```diff
@@ -59,6 +59,27 @@ typedef unsigned __int64	uint64_t;
 
 #include <stdlib.h>
 
+// 1.cpp:4:12: warning: duplicate 'extern' declaration specifier
+//      [-Wduplicate-decl-specifier]
+// extern "C" extern int c;
+//
+// Therefore, remove extern on functions as always meaningless/redundant,
+// and provide MONO_API_DATA for data, that always has one and only one extern.
+
+#ifdef __cplusplus
```

Comment:
You should define a `MONO_EXTERN_C extern "C"` and use that at https://github.com/mono/mono/pull/9814/files#diff-bd8dbfad6e5c0280586ab06f8513939dR84 instead to avoid repetition.

Reply:
Yes I considered that but didn't want to add to the public symbol surface area. Ok. Will do.